elks/tools/elf2elks/elf2elks.c

```diff
@@ -243,6 +248,12 @@ parse_args (int argc, char **argv)
 		tiny = true;
 	      else if (strcmp (arg + 2, "symtab") == 0)
 		symtab = true;
+	      else if (strcmp (arg + 2, "symfile") == 0)
+		{
+		    symfile = true;
+		    ++i;
+		    strcpy(symtab_filename, argv[i]);
```

Comment:
Minor nit: You might want to check that `i < argc` still, i.e. there is actually a file name after `--symfile`.

Reply:
Thank you, fixed with error message.